### DISCUSSION TITLE
Zugriff Weboberfläche

### DISCUSSION THREAD

**Discussion by mak-86** [brutella/hkknx-public#174] [Q&A] [answered]
Guten Morgen,

ich versuche aktuell die Weboberfläche aufzurufen. Ich habe die vorgesehene Datei installiert komme aber mit Benutzername.local:8080 oder IP-Adresse:8080 nicht drauf. Habe es zum Testen erstmal auf dem Mac installiert. Raspberry ist ja leider überall ausverkauft.

Vielleicht irgendeine Idee was ich falsch mache?

Vielen Dank im Voraus!

Viele Grüße

Marcel

**Comment by brutella** [comment, marked as answer]
Wenn hkknx auf deinem Mac läuft, dann kannst du mit http://localhost:8080 von deinem Mac aus darauf zugreifen.

**Reply by mak-86** [reply]
Vielen Dank für deine schnelle Antwort. 
Ich habe die Datei "hkknx" gestartet, aber mit http://localhost:8080/ kann er keine Verbindung aufbauen.

**Comment by brutella** [comment]
Zeig mal einen Screenshot vom Terminal wie du hkknx startest. Was wird da angezeigt?

–Matthias

**Reply by mak-86** [reply]
Ich merke gerade, dass da das Problem liegt. Ich habe einfach die Datei heruntergeladen und ausgeführt. Ich schau nochmal wie ich es starten kann... Danke

**Comment by brutella** [comment]
Konntest du das Problem schon lösen?

**Reply by brutella** [reply]
Das Ausführen von `hkknx` ist auf dem Mac nichts anderes. Lese dir dazu das [Online Handbuch](https://hochgatterer.me/de/hkknx/docs/install/rpi-cli/#installation) genau durch.

**Comment by Nathanael2316** [comment]
Ich hänge mich mal hier mit ran.
MacBook Pro aus 2012 gepatched. Ventura 13.2.1
Ich starte die "hkknx" mit doppelklick. Gehöre zu den Anfängern :-)
Im Terminal steht folgendes:
Last login: Wed Feb 15 20:52:35 on ttys000
/Users/nathanael/Downloads/hkknx-2.5.0_darwin_amd64/hkknx ; exit;
nathanael@ Nathanaels-MacBook-Pro ~ % /Users/nathanael/Downloads/hkknx-2.5.0_darwin_amd64/hkknx ; exit;
INFO 2023/02/15 20:52:49 main.go:79: version 2.5.0 (built at 2023-02-01T16:40:17Z+0100)
  -autoupdate
    	Auto updating the app every 24 hours. (default true)
  -db string
    	Path to database folder
  -port int
    	Port of the server (default 8080)
  -verbose
    	Enable verbose logging
  -version
    	Prints the version number

Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Prozess beendet]
